src/Controllers/DatabaseController.js

```
@@ -132,23 +132,23 @@ const validateQuery = (
   }
 
   if (query.$or) {
-    if (query.$or instanceof Array) {
+    if (Array.isArray(query.$or)) {
       query.$or.forEach(value => validateQuery(value, isMaster, isMaintenance, update, options, _depth + 1));
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)